apps/web/pages/api/users/username/[username].ts

```diff
@@ -0,0 +1,51 @@
+import type { NextApiRequest, NextApiResponse } from "next";
```

Comment:
This new endpoint appears to be publicly accessible without any rate limiting. Consider adding rate limiting to prevent potential enumeration attacks or DoS attempts. You might want to use a solution like `@calcom/lib/rateLimit` to protect this endpoint, especially since it can be used to enumerate usernames on the platform.